common/Perf/Azure.Test.Perf/PerfProgram.cs

```diff
@@ -68,19 +68,6 @@ private static async Task Run(Type testType, PerfOptions options)
                 Console.WriteLine("Application started.");
             }
 
-            Console.WriteLine("=== Versions ===");
```

Comment:
It may be both interesting and valuable to keep this and print both.   I wonder if seeing the difference between referenced assemblies and those actually loaded may both help us understand if our references can be trimmed and help to more quickly identify discrepancies between local/hosted runs.

Reply:
I agree it's good to verify that referenced matches loaded, so I added the referenced version to the output.  However, `Assembly.GetReferencedAssemblies()` cannot be used to find unused references in csproj, since they are already removed by the compiler:

https://stackoverflow.com/a/3434002/3630403